landing/templates/landing/index.html

```diff
@@ -1,8 +1,59 @@
-{% extends 'base.html' %}
```

Comment:
Why do you remove the base.html and instead re-define the entire HTML page here?